boards/arm/cc3235sf_launchxl/doc/index.rst

```diff
@@ -0,0 +1,305 @@
+   with CMSIS-DAP" appearing at /dev/ttyACM1 and /dev/ttyACM0.
+
+#. Update the service pack, and place the board in "Development Mode".
+
+   Setting "Development Mode" enables the JTAG interface, necessary
+   for subsequent use of OpenOCD and updating XDS110 firmware.
+
+   Follow the instructions in Section 2.4 "Download the Application",
+   in the `CC32xx Quick Start Guide`_, except for steps 5 and 6 in
+   Section 2.4.1 which select an MCU image.
+
+#. Ensure the XDS-110 emulation firmware is updated.
+
+   Download and install the latest `XDS-110 emulation package`_.
+
+   Follow the directions here to update the firmware:
+   http://software-dl.ti.com/ccs/esd/documents/xdsdebugprobes/emu_xds110.html#updating-the-xds110-firmware
+
+   Note that the emulation package install may place the xdsdfu utility
+   in <install_dir>/ccs_base/common/uscif/xds110/.
+
+#. Switch Jumper SOP[2..0] (J15) back to [001].
+
+   Remove power from the board (disconnect USB cable) before switching jumpers.
+
+#. Install TI OpenOCD
+
+   Clone the TI OpenOCD git repository from: http://git.ti.com/sdo-emu/openocd.
+   Follow the instructions in the Release Notes in that repository to build
+   and install.
+
+   Since the default TI OpenOCD installation is /usr/local/bin/,
+   and /usr/local/share/, you may want to backup any current openocd
+   installations there.
+   If you decide to change the default installation location, also update
+   the OPENOCD path variable in :zephyr_file:`boards/arm/cc3235sf_launchxl/board.cmake`.
+
+#. Ensure CONFIG_XIP=y (default) is set.
+
+   This locates the program into flash, and sets CONFIG_CC3235SF_DEBUG=y,
+   which prepends a debug header enabling the flash to persist over
+   subsequent reboots, bypassing the bootloader flash signature
+   verification.
+
+   See Section 21.10 "Debugging Flash User Application Using JTAG" of the
+   `CC3235 TRM`_ for details on the secure flash boot process.
+
+
+Once the above prerequisites are met, applications for the ``_cc3235sf_launchxl``
+board can be built, flashed, and debugged with openocd and gdb per the Zephyr
+Application Development Primer (see :ref:`build_an_application` and
+:ref:`application_run`).
+
+Flashing
+========
+
+To build and flash an application, execute the following commands for <my_app>:
+
+.. zephyr-app-commands::
+   :zephyr-app: <my_app>
+   :board: cc3235sf_launchxl
+   :goals: flash
+
+This will load the image into flash.
+
+To see program output from UART0, connect a separate terminal window:
+
+.. code-block:: console
+
+  % screen /dev/ttyACM0 115200 8N1
+
+Then press the reset button (SW1) on the board to run the program.
+
+Debugging
+=========
+
+To debug a previously flashed image, after resetting the board, use the 'debug'
+build target:
+
+.. zephyr-app-commands::
+   :zephyr-app: <my_app>
+   :board: cc3235sf_launchxl
+   :maybe-skip-config:
+   :goals: debug
+
+
+WiFi Support
```

Comment:
Wi-Fi is the correct spelling of the trademarked name. Fix all text references (there are few more WiFi uses below).